samples/bluetooth/peripheral_hids_keyboard/sample.yaml

```diff
@@ -22,13 +22,17 @@ tests:
       - nrf52840dk/nrf52840
       - nrf5340dk/nrf5340/cpuapp
       - nrf5340dk/nrf5340/cpuapp/ns
+      - nrf54l15dk/nrf54l05/cpuapp
+      - nrf54l15dk/nrf54l10/cpuapp
       - nrf54l15dk/nrf54l15/cpuapp
       - nrf54h20dk/nrf54h20/cpuapp
     platform_allow:
       - nrf52dk/nrf52832
       - nrf52840dk/nrf52840
       - nrf5340dk/nrf5340/cpuapp
       - nrf5340dk/nrf5340/cpuapp/ns
+      - nrf54l15dk/nrf54l05/cpuapp
```

Comment:
Can we add e.g. L15 and L10 configurations to integration quarantine to limit CI usage? (same for the central HIDS sample). nRF54L boards should act similarly (the only difference is available memory)

Reply:
That's a good idea. I will add this change.